java/README.md

```diff
@@ -29,27 +30,49 @@ Software Dependencies:
 - protoc (protobuf compiler)
 - Rust
 
+Please also consider installing the following packages to build [GLIDE core rust library](../glide-core/README.md):
```

Comment:
should we just reference the glide-core library here as a dependency? 
[GLIDE core library](../glide-core/README.md)

Reply:
We can, but the readme of core is empty. TBD to copy this block to that readme.

Reply:
See  #113